Composer/packages/extension/package.json

```diff
@@ -25,6 +24,7 @@
   },
   "dependencies": {
     "@botframework-composer/types": "*",
+    "@types/passport": "^1.0.3",
```

Comment:
it's a dev dep right?

Reply:
This actually needs to be in the dependencies because we reference it in an exported type.